docs/source/installation.md

```diff
@@ -18,16 +18,17 @@ issues when using multiple conda channels (e.g. defaults and conda-forge).
 
 ## JAX sampling
 
-If you wish to enable sampling using the JAX backend via NumPyro, the following should also be installed:
+If you wish to enable sampling using the JAX backend via NumPyro,
+you need to install it manually as it is an optional dependency:
 
 ```console
-pip install numpyro
+conda install numpyro
 ```
 
-Similarly, to use BlackJAX for sampling it should be installed via `pip`:
+Similarly, to use BlackJAX sampler instead:
 
 ```console
-pip install blackjax
+conda install blackjax
```

Comment:
is blackjax and numpyro on conda? Asking because JAX is not and must be pip installed?

Reply:
They all are: https://anaconda.org/conda-forge/jax, and our conda envs already install them from conda: https://github.com/pymc-devs/pymc/blob/main/conda-envs/environment-docs.yml#L21

Reply:
Cool, didn't know